### PR TITLE
guide(@akarui/aoi.music v5)

### PR DESCRIPTION
**Pull request by USERSATOSHI**
aoi.music guide

### CODE REVIEW

**Comment by ishh-xd** [comment]
Lel

**Comment by Leref** [comment]
Keep in mind, since it's using Gitbook, it'll not appear as its in sync using SUMMARY.MD for its table of contents placement,
Please add it the according position.

**Comment by Leref** [comment]
Keep in mind, since it's using Gitbook, it'll not appear as its in sync using `SUMMARY.md` for its table of contents placement,
Please add into the according position.

File: src/guide/extras/music.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Music
+
+aoi.js has an extension named ***"@akarui/aoi.music"*** for music.
+
+## installation
```
Comment (Leref) [inline src/guide/extras/music.md]: Proper capitalization.

File: src/guide/extras/music.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+voice.onTrackStart();
+
+loader.load(bot.cmd, "./Commands/commands/"); //bot cmds
+loader.load(voice.cmd, "./Commands/voice/"); //voice cmds
+```
+
+## using aoi.music
+
+as we got a setup in index.js ,now let's create some cmds
```
Comment (Leref) [inline src/guide/extras/music.md]: This is a poor written explanation.

File: src/guide/extras/music.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+### play
+
+
+
```
Comment (Leref) [inline src/guide/extras/music.md:64]: This formatting is need of work,
So many indents for no reason.

File: src/guide/extras/music.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+## installation
+
+> ```
+> npm i @akarui/aoi.music
+> ```
+
+***for Dev Version***
+
+> ```
+> npm i https://github.com/akaruidevelopment/music#main
```
Comment (Leref) [inline src/guide/extras/music.md]: @dev for @akarui/aoi.music will be soon.

File: src/guide/extras/music.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+> ```
+
+### setting up in aoi.js
```
Comment (Leref) [inline src/guide/extras/music.md]: Can we properly have uppercase.

File: src/SUMMARY.md [hunk 6]
```diff
@@ -26,6 +26,7 @@
     * [Slash Commands](guide/advanced-guides/slash-commands.md)
   * [Extras](guide/extras/README.md)
     * [Custom Events](guide/extras/custom-events.md)
+    * [Voice](guide/extras/music.md)
```
Comment (Leref) [inline src/SUMMARY.md:29]: This should be @akarui/aoi.music, since this entire guide is related towards it.

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/guide/extras/music.md (hunk 1 above)
Comment (USERSATOSHI) [reply src/guide/extras/music.md]: for line 3 or line 5?

File: src/guide/extras/music.md (hunk 2 above)
Comment (USERSATOSHI) [reply src/guide/extras/music.md]: there wasn't much to explain this line , but this change should do

File: src/guide/extras/music.md (hunk 4 above)
Comment (USERSATOSHI) [reply src/guide/extras/music.md]: ok done

File: src/guide/extras/music.md (hunk 3 above)
Comment (USERSATOSHI) [reply src/guide/extras/music.md:64]: removed extra space

File: src/guide/extras/music.md (hunk 1 above)
Comment (USERSATOSHI) [reply src/guide/extras/music.md]: line 5 done

File: src/SUMMARY.md (hunk 6 above)
Comment (USERSATOSHI) [reply src/SUMMARY.md:29]: it's  more related to Voice class cause if i add aoi.music then i have to explain all classes

**Review by Leref — APPROVED** [review]

**Pull request merged**